packages/base/server.js

```diff
@@ -124,6 +125,16 @@ const start = async (
   server = express()
   httpServer = createServer(server)
 
+  server.use(helmet({
+    hsts: {
+      maxAge: 60 * 60 * 24, // one day for initial testing
+      // maxAge: 31536000, // 1 year to get preload approval
+      // preload: true,
+      includeSubDomains: true
+    },
+    referrerPolicy: true
```

Comment:
For the sake of clarity would set explicit (albeit [default](https://helmetjs.github.io/docs/referrer-policy/)) value: `no-referrer`